### PR TITLE
Refs #28028 - update chromedriver to match jenkins version

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Review by tbrisker — APPROVED** [review]
Thanks @mmoll !

**Comment by tbrisker** [comment]
1.24 - 35db88a4a
1.23 - cefa40b49

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "react-intl": "^2.8.0"
   },
   "optionalDependencies": {
-    "chromedriver": "77.0.0"
+    "chromedriver": "79.0.0"
```
Comment (ekohl) [inline package.json:29]: Since we always update to the latest, should we just make it `>= 79`?

Comment (mmoll) [reply package.json:29]: I think this is not really better, as the relevant version is (AFAIK) the chromium version of EPEL, which seems not to update every version.

**Pull request merged**